homeassistant/components/doorbird.py

```diff
@@ -239,6 +248,13 @@ def _register_event(self, hass_url, event, schedule):
                 entry = self.device.get_schedule_entry(event, str(doorbell))
                 entry.output.append(output)
                 self.device.change_schedule(entry)
+        elif event == 'relay':
+            # Repeat edit for each monitored doorbell number
+            for relay in self._relay_nums:
+                entry = self.device.get_schedule_entry(event, str(relay))
+                entry.output.append(output)
+                resp = self.device.change_schedule(entry)
+                return resp
```

Comment:
Is this return correct? It looks weird in this place. Also, nothing is checking the return value of this method.

Reply:
The return is unnecessary, but shouldn't cause any problems since the calling code does not expect a return value.  I have some other housekeeping tasks planned for this component and will clean this up as part of those changes.

Reply:
Should we really return inside the loop?

Reply:
Nope.  You're right.  Guess I'll go ahead and knock out those changes...